From 0065085ebd1fe3f755113167c03c55ed08727fa3 Mon Sep 17 00:00:00 2001
From: Yuxi Sun <b36102@freescale.com>
Date: Fri, 24 Aug 2012 15:55:25 +0800
Subject: [PATCH 344/505] ENGR00221348 v4l2 autorun script: change default
 camera to ov5642

Default to load ov5642 modules when run autorun-v4l2.sh

Signed-off-by: Yuxi Sun <b36102@freescale.com>
---
 test/mxc_v4l2_test/autorun-v4l2.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/mxc_v4l2_test/autorun-v4l2.sh b/test/mxc_v4l2_test/autorun-v4l2.sh
index 2322094..bd8b25a 100755
--- a/test/mxc_v4l2_test/autorun-v4l2.sh
+++ b/test/mxc_v4l2_test/autorun-v4l2.sh
@@ -32,7 +32,7 @@ if [ $TEST_CAMERA = 1 ]; then
 			modprobe ov5642_camera
 		fi
 		if [ "$(platform)" = "IMX6" ]; then
-			modprobe ov5640_camera
+			modprobe ov5642_camera
 		fi
 		modprobe mxc_v4l2_capture
 	fi
-- 
1.8.0

